cg/constants/pacbio.py

```diff
@@ -16,6 +17,15 @@ class PacBioDirsAndFiles:
     RUN_IS_VALID: str = "is_valid"
 
 
+class MetricsFileFields:
+    ATTRIBUTES: str = "attributes"
+    COLUMNS: str = "columns"
+    ID: str = "id"
+    TABLES: str = "tables"
+    VALUE: str = "value"
+    VALUES: str = "values"
+
+
 class CCSAttributeIDs:
```

Comment:
lets standardise the name sof these classes. I liked the `Alias` suffix that @islean used

Reply:
fixed